pkg/toplevel/server/toplevel-impl.go

```diff
@@ -100,6 +100,8 @@ func (i *TopLevelServer) grpcGetTransactions(ctx context.Context) (*externalRef0
 
 func convertTrasaction(networkChange *admin.ListTransactionsResponse) externalRef0.Transaction {
 
+	log.Infof("TEO Transaction: %v", networkChange.Transaction)
```

Comment:
Nice!

Reply:
I might want to remove that before merging :)